app/ante/eth.go

```diff
@@ -41,43 +42,35 @@ import (
 // - from address is empty
 // - account balance is lower than the transaction cost
 func VerifyEthAccount(
-	ctx sdk.Context, tx sdk.Tx,
+	ctx sdk.Context,
+	msgEthTx *evmtypes.MsgEthereumTx,
+	tx *ethtypes.Transaction,
```

Comment:
it's not nesserary to manually cache this field I think, since it's simply `msgEthTx.Raw.Transaction`.